ach.gemspec

```
@@ -1,22 +1,33 @@
 # -*- encoding: utf-8 -*-
 $:.push File.expand_path("../lib", __FILE__)
-require 'ach/version'
+require "ach/version"
 
 Gem::Specification.new do |s|
-  s.name        = 'ach'
+  s.name        = "ach"
   s.version     = ACH::VERSION.dup
-  s.summary     = 'Helper for building ACH files'
-  s.description = 'ach is a Ruby helper for building and parsing ACH files. In particular, it helps with field order and alignment, and adds padding lines to end of file.'
-  s.email       = 'jmorgan@morgancreative.net'
-  s.homepage    = 'https://github.com/jm81/ach'
-  s.authors     = ['Jared Morgan', 'Josh Puetz']
+  s.summary     = "Helper for building ACH files"
+  s.description = "ach is a Ruby helper for building and parsing ACH files. In particular, it helps with field order and alignment, and adds padding lines to end of file."
+  s.email       = "jmorgan@morgancreative.net"
```

Please change this to my email:
```suggestion
  s.email       = "henriquegasques@gmail.com"
```